web/themes/custom/recipe_theme/templates/field/field--node--field-ingredients.html.twig

```diff
@@ -0,0 +1,52 @@
+{#
+/**
+ * @file
+ * Default template for a field.
+ */
+#}
+{%
+  set classes = [
+  bundle|clean_class ~ '__' ~ field_name_clean|clean_class,
+  label_display == 'inline' ? 'd-flex',
+]
+%}
+{%
+  set title_classes = [
+  'field__label',
+  'font-weight-bold',
+  'field-ingredints',
```

Comment:
I don't think you need to add this class (also, it has a typo).

The theme (probably Radix) provides some classes out of the box

![Screenshot_20200330_152305](https://user-images.githubusercontent.com/13573052/77917190-89c2da80-729a-11ea-879f-aad151684ef9.png)